finagle-thrift/src/main/scala/com/twitter/finagle/rich.scala

```diff
@@ -145,7 +146,8 @@ private[twitter] object ThriftUtil {
     impl: AnyRef,
     protocolFactory: TProtocolFactory,
     stats: StatsReceiver,
-    maxThriftBufferSize: Int
+    maxThriftBufferSize: Int,
+    label: String
```

Comment:
hmm, we don't want to break binary backcompat.  can you overload with the old method too?